server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/dash/sources/coinmarketcap/ICoinmarketcapAPI.java

```diff
@@ -21,12 +21,13 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.core.MediaType;
 
 @Path("/v1/ticker")
 @Produces(MediaType.APPLICATION_JSON)
 public interface ICoinmarketcapAPI {
     @GET
-    @Path("/")
-    CMCTicker[]  getTickers(@QueryParam("convert") String fiatCurrency);
+    @Path("/{cryptoCurrency}/")
```

Comment:
Revert to the original way - cryptoCurrency parameter is the coin symbol, not the full name. CMC API uses full names unfortunately. This wouldn't work.